src/Toolbar/ToolbarTitle.js

```diff
@@ -25,6 +25,10 @@ class ToolbarTitle extends Component {
 
   static propTypes = {
     /**
+     * The component(s) to be displayed.
+     */
+    children: PropTypes.node,
```

Comment:
I know, it's sad that we have named the property `text` over `children` but that `master` branch is only on a *going to die* maintenance mode. Please only stick to the bug and not introduce a new API. By the way, we are much more coherent with the API on the `next` branch.